src/Test/Perf/infra/MonitorShareTask.bat

```diff
@@ -0,0 +1,11 @@
+@echo off
```

Comment:
We need to add a scheduled task to the machine which runs the MonitorShareTask with the share to monitor as the argument